package.json

```diff
@@ -38,91 +38,87 @@
+    "qs": "^6.11.0",
+    "react": "^18.2.0",
     "react-dnd": "^5.0.0",
     "react-dnd-html5-backend": "^5.0.1",
-    "react-dom": "^16.6.1",
+    "react-dom": "^18.2.0",
     "react-load-script": "^0.0.6",
-    "react-redux": "^5.0.7",
-    "react-router": "^4.4.0-beta.6",
-    "react-router-config": "^4.4.0-beta.6",
-    "react-router-dom": "^4.4.0-beta.6",
-    "react-select": "^1.3",
-    "reactstrap": "^8.5.1",
-    "redux": "^4.0.0",
-    "redux-form": "^7.4.2",
-    "redux-thunk": "^2.1.0",
-    "resize-observer-polyfill": "^1.5.0",
-    "tinymce": "^6",
-    "toposort": "^1.0.3",
+    "react-redux": "^8.0.4",
+    "react-router": "^6.4.2",
+    "react-router-dom": "^6.4.2",
+    "react-select": "^5.5.8",
+    "reactstrap": "^8.9.0",
+    "redux": "^4.2.0",
+    "redux-form": "^8.3.8",
+    "redux-thunk": "^2.4.1",
+    "remove-files-webpack-plugin": "^1.5.0",
+    "resize-observer-polyfill": "^1.5.1",
+    "tinymce": "^6.2.0",
+    "toposort": "^2.0.2",
     "url": "^0.11.0",
-    "uuid": "^8.2.0",
-    "validator": "^6.1.0",
-    "webpack-sources": "^1.1.0"
+    "uuid": "^9.0.0",
+    "validator": "^13.7.0",
+    "webpack-sources": "^3.2.3"
   },
   "devDependencies": {
-    "@silverstripe/eslint-config": "^0.1.0",
-    "@silverstripe/webpack-config": "^1.7.0",
-    "@storybook/addon-actions": "^3.4.10",
-    "@storybook/addon-knobs": "^3.4.10",
-    "@storybook/addon-notes": "^3.4.10",
-    "@storybook/addon-options": "^3.4.10",
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "babel-jest": "^23.6.0",
-    "babel-runtime": "^6.26.0",
-    "chalk": "^2.4.2",
-    "copy-webpack-plugin": "^4.2.0",
-    "enzyme": "^3.6.0",
-    "enzyme-adapter-react-16": "^1.5.0",
-    "expose-loader": "^0.7.3",
-    "fast-glob": "^3.0.1",
-    "html-loader": "^0.5.1",
+    "@babel/runtime": "^7.20.0",
+    "@silverstripe/eslint-config": "^1.0.0-alpha1",
+    "@silverstripe/webpack-config": "^2.0.0-alpha1",
+    "babel-jest": "^29.2.2",
+    "chalk": "^5.1.2",
+    "copy-webpack-plugin": "^11.0.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
+    "fast-glob": "^3.2.12",
+    "html-loader": "^4.2.0",
     "is-plain-object": "^5.0.0",
-    "jest-cli": "^23.6.0",
-    "markdown-loader": "^5.1.0",
-    "sass-loader": "^6.0.5",
-    "storybook-addon-jsx": "^5.4.0",
-    "webpack": "^3.12.0"
+    "jest-cli": "^29.2.2",
+    "jest-environment-jsdom": "^29.3.1",
+    "markdown-loader": "^8.0.0",
+    "react-16": "npm:react@^16.14.0",
+    "react-dom-16": "npm:react-dom@^16.14.0",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^5.0.0"
   },
   "resolutions": {
-    "colors": "1.1.2",
-    "eslint": "^4.6.1"
+    "colors": "1.4.0"
   },
+  "browserslist": [
+    "defaults"
+  ],
   "jest": {
+    "testEnvironment": "jsdom",
+    "moduleNameMapper": {
+      "^react-dom/client$": "react-dom-16",
+      "^react-dom((/.*)?)$": "react-dom-16$1",
+      "^react((/.*)?)$": "react-16$1"
+    },
```

Comment:
This is a temporary _hacky_ workaround for https://github.com/silverstripe/silverstripe-admin/issues/1419